src/main/resources/templates/postCreationSuccess.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Just a quick change. The following [Profile Use Case PR](https://github.com/CSC207-2022F-UofT/course-project-fitbook/pull/46/files#top) has the updated profile.html page with a nav section and respective styling. 

Could you just copy that appropriately into this html file so the user can leave the success page?

Reply:
got it, file being updated